bgpd/bgp_community.c

```diff
@@ -308,12 +308,11 @@ static void set_community_string(struct community *com, bool make_json)
 		if (first)
 			first = 0;
 		else
-			*pnt++ = ' ';
+			strlcat(str, " ", len);
 
 		switch (comval) {
 		case COMMUNITY_INTERNET:
-			strcpy(pnt, "internet");
-			pnt += strlen("internet");
+			strlcat(str, "internet", len);
```

Comment:
using str and pnt interchangeably may confuse. will it be a good idea to remove one of this?

Reply:
Yep